rails_app/app/assets/javascripts/application.js

```diff
@@ -13,3 +13,5 @@
 //= require rails-ujs
 //= require turbolinks
 //= require_tree .
+//= require jquery
```

Comment:
Do we really need jQuery explicitly?
You might want to read this: https://css-tricks.com/now-ever-might-not-need-jquery/

Reply:
Bootstrap JavaScript depends on jQuery. If you're using Rails 5.1+, add the jquery-rails gem to your Gemfile:

gem 'jquery-rails'